carefull-online_myproj_i2_18.py

```
@@ -0,0 +1,161 @@
+weights_path="mask_rcnn_damage_0010.h5"
+model = modellib.MaskRCNN(mode="inference", config=config,
+                                  model_dir=DEFAULT_LOGS_DIR)
+model.load_weights(weights_path, by_name=True)
+
+
+
+
+
+def detect_and_color_splash(model, image_path=None, video_path=None):
+    assert image_path or video_path
+    class_names = ['damage']
```

Ruff (0.13.1)

Local variable `class_names` is assigned to but never used

Remove assignment to unused variable `class_names`

(F841)